core/synch.c

```diff
@@ -2248,7 +2248,7 @@ detach_on_permanent_stack(bool internal, bool do_cleanup, dr_stats_t *drstats)
         dynamo_thread_exit_pre_client(my_dcontext, my_tr->id);
     }
 
-    LOG(GLOBAL, LOG_ALL, 1, "Detach: Letting slave threads go native\n");
+    LOG(GLOBAL, LOG_ALL, 1, "Detach: Letting worker threads go native\n");
```

Comment:
Like below, I would replace this with whatever is chosen for detach there: "secondary" or "follower"

Reply:
Done; changed to "secondary"